package.json

```diff
@@ -6,6 +6,9 @@
   "repository": "muut/riotjs",
   "author": "Muut, Inc. and other contributors",
   "license": "MIT",
+  "scripts": {
+    "test": "make test"
```

Comment:
Why invoke make from node when you're just running `./node_modules/mocha/bin/mocha test/runner.js` anyway. Seems like unneeded indirection